DUI3-DX/Converters/Rhino/Speckle.Converters.Rhino7.DependencyInjection/Speckle.Converters.Rhino7.DependencyInjection.csproj

```diff
@@ -1,19 +1,18 @@
 <Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net48</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <CopyLocalSatelliteAssemblies>false</CopyLocalSatelliteAssemblies>
 
-    <PropertyGroup>
-        <TargetFramework>net48</TargetFramework>
-        <ImplicitUsings>enable</ImplicitUsings>
-        <Nullable>enable</Nullable>
-    </PropertyGroup>
+  </PropertyGroup>
 
-    <ItemGroup>
-      <ProjectReference Include="..\..\..\Sdk\Speckle.Autofac\Speckle.Autofac.csproj" />
-      <ProjectReference Include="..\..\..\Sdk\Speckle.Converters.Common\Speckle.Converters.Common.csproj" />
-      <ProjectReference Include="..\..\..\Converters\Rhino\Speckle.Converters.Rhino7\Speckle.Converters.Rhino7.csproj" />
-    </ItemGroup>
-
-    <ItemGroup>
-      <PackageReference Include="RhinoCommon" Version="7.13.21348.13001" />
-    </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\Sdk\Speckle.Converters.Common.DependencyInjection\Speckle.Converters.Common.DependencyInjection.csproj" PrivateAssets="All"/>
+    <ProjectReference Include="..\..\..\Converters\Rhino\Speckle.Converters.Rhino7\Speckle.Converters.Rhino7.csproj" PrivateAssets="All" />
+  </ItemGroup>
```

Comment:
I'm going to suggest this be the pattern we follow, at least for now, to prevent converters from being exposed to the connector. Only the converter module will.

To do this, we set all direct dependencies of Converters.*.DependencyInjection projects to be `PrivateAssets="All"`